test/jdk/java/nio/file/attribute/BasicFileAttributeView/CreationTime.java

```diff
@@ -88,6 +88,12 @@ static void test(Path top) throws IOException {
                 supportsCreationTimeRead = true;
                 supportsCreationTimeWrite = true;
             }
+        } else if (Platform.isLinux()) {
+            // Creation time read depends on statx system call support
+            supportsCreationTimeRead = CreationTimeHelper.linuxIsCreationTimeSupported();
+            System.out.println("[Linux] Debug: supportsCreationTimeRead == " + supportsCreationTimeRead);
+            // Creation time updates are not supported on Linux
+            supportsCreationTimeWrite = false;
```

Comment:
It might be simpler to use `Linker.nativeLinker().defaultLookup().find("statx").isPresent()` as that would avoid adding CreationTimeHelper. JEP 454 is proposed to target 22 so the test won't need @enablePreview.

Reply:
OK. If I use that this patch will have to wait for https://github.com/openjdk/jdk/pull/15103 to get integrated before this can be merged (avoiding the need to add `@enablePreview`). But that should happen fairly soon so it should be fine.

Reply:
For now I'm still using `@enablePreview` and will drop before integration once it's ready.